tests/test_clients.py

```diff
@@ -632,3 +632,19 @@ async def test_crossref_retraction_status(stub_data_dir: Path) -> None:
 def test_reformat_name(name: str, expected: str) -> None:
     result = reformat_name(name)
     assert result == expected, f"Expected '{expected}', but got '{result}' for '{name}'"
+
+
+@pytest.mark.vcr
+@pytest.mark.asyncio
+async def test_arxiv_dois() -> None:
```

Comment:
Can you make this test name a bit more verbose/self-explanatory?

Something like `test_arxiv_doi_is_used_when_available`